src/explore-education-statistics-common/src/components/PageNav.tsx

```diff
@@ -0,0 +1,105 @@
+import styles from '@common/components/PageNav.module.scss';
```

Comment:
Can the existing nav when viewing release contents on the public site be changed to use these common components at some point, or are they different beasts altogether?

Reply:
The one on the new data catalogue details page can probably use this and maybe other places, I didn't want to go far down a refactoring rabbit hole though so have left if for now.